opp_area_selection/config/area_selection_parameters.yaml

```diff
@@ -0,0 +1,8 @@
+cluster_tolerance: 0.25
```

Comment:
It'd be nice to have some comments next to the parameters to briefly explain what they do, I'm ok with creating an issue to address this at a later time

Reply:
See #3